pkgs/tools/misc/digitemp/default.nix

```diff
@@ -0,0 +1,68 @@
+{ fetchFromGitHub
+, gnumake
+, lib
+, libusb
+, stdenv
+
+, defaultAdapter ? "DS9097U"
+# that support seems to be broken
+, ds2490 ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "digitemp";
+  version = "3.7.2";
+
+  src = fetchFromGitHub {
+    owner = "bcl";
+    repo = "digitemp";
+    rev = "v${version}";
+    sha256 = "19zka5fcdxhhginaspak76l984iqq9v2j6qrwvi5mvca7bcj8f72";
+  };
+
+  nativeBuildInputs = [ gnumake ];
+  buildInputs = lib.optional ds2490 libusb;
+
+  buildPhase = ''
+    runHook preBuild
+    make clean
+    make -j$NIX_BUILD_CORES LOCK=no ds9097 ds9097u ${lib.optionalString ds2490 "ds2490"}
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -D -t $out/bin digitemp_*
+  '' + (lib.optionalString (defaultAdapter != "") ''
+    pushd $out/bin
+    ln -s "digitemp_${defaultAdapter}" digitemp
+    popd
```

Comment:
maybe I should split this part out into another derivation.

Reply:
I got rid of it. I probably won't add it back, as it is almost trivial to create a symlink derivation.